### ISSUE TITLE
Visualization notebook is not running

### ISSUE THREAD

**Issue by Bisuketto** [fmfn/BayesianOptimization#350]
The visualization.ipynb in the examples notebook doesn't run.
I'm also not able to run the optimization on a 1 parameter function. The two errors seem to be linked.

Using : python=3.10.4, scipy=1.9.0, numpy=1.23.1 and this repo commit [7e888d7]

The error i got when trying to run maximize on a 1 parameter function : 
```
Traceback (most recent call last):
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/bayes_opt/bayesian_optimization.py", line 240, in maximize
    x_probe = self._queue.get(block=False)
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/queue.py", line 168, in get
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/xxx/Documents/bayes-test/visu.py", line 46, in <module>
    optimizer.maximize(
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/bayes_opt/bayesian_optimization.py", line 243, in maximize
    x_probe = self.suggest(util)
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/bayes_opt/bayesian_optimization.py", line 157, in suggest
    suggestion = acq_max(
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/bayes_opt/util.py", line 55, in acq_max
    res = minimize(lambda x: -ac(x.reshape(1, -1), gp=gp, y_max=y_max),
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/scipy/optimize/_minimize.py", line 684, in minimize
    bounds = standardize_bounds(bounds, x0, meth)
  File "/home/xxx/miniconda3/envs/bayes-opt/lib/python3.10/site-packages/scipy/optimize/_minimize.py", line 947, in standardize_bounds
    bounds = new_bounds_to_old(bounds.lb, bounds.ub, x0.shape[0])
IndexError: tuple index out of range
```

Minimal code to produce this error : 
``` python
from bayes_opt import BayesianOptimization

def target(x):
    return x*3
pbounds = {'x': (-10, 10)}
optimizer = BayesianOptimization(
    f=target,
    pbounds=pbounds,
    verbose=2,
    random_state=1,
)
optimizer.maximize(
    init_points=2,
    n_iter=10,
)
```

**Comment by bwheelz36** [comment]
Hey, thanks for finding this. 

This example runs fine with scipy 1.8.1, but fails with scipy 1.9.0.
The issue is that in the new version scipy seems to be changing the way they want to define bounds. I'm going to check on the scipy page whether they consider this a bug or not.

**Comment by bwheelz36** [comment]
I made a PR to fix this in #351 
- made the change suggested above
- added this minimal example to tests so we catch such cases in the future
Would appreciate if anyone has time to review.
